spec/unleash/toggle_fetcher_spec.rb

```diff
@@ -52,4 +58,31 @@
       end
     end
   end
+
+  describe '.new' do
+    context 'when there are problems fetching toggles' do
+      before do
+        cache_creator = described_class.new
+        cache_creator.toggle_cache = { features: [] }
+        cache_creator.save!
+
+        WebMock.stub_request(:get, "http://toggle-fetcher-test-url/client/features").to_return(status: 500)
+      end
+
+      it 'reads the backup file for values' do
+        expect(toggle_fetcher.toggle_cache).to eq("features" => [])
```

Comment:
Nitpick: Symbols vs Strings again 😄 
```suggestion
        expect(toggle_fetcher.toggle_cache).to eq(features: [])
```

Reply:
`JSON.parse` uses strings as keys unless the `symbolize_names` parameter is given to it. Using symbols as keys here would fail the expectation.

Reply:
Good point!